docs/design/autoware-architecture/localization/index.md

```diff
@@ -1,5 +1,294 @@
-# Localization component design
+LOCALIZATION COMPONENT DESIGN DOC
 
-!!! warning
+## Abstract
 
-    Under Construction
+## 1. Requirements
+
+Localization aims to estimate vehicle pose, velocity, and acceleration.
+
+Goals:
+
+- Proposition of a system that can estimate the vehicle pose, velocity, and acceleration as long as possible
+- Proposition of a system that can diagnose the stability of estimation and send a warning message to the error-monitoring system if the estimation result is unreliable
+- Vehicle localization function can adequately work with various sensor configurations
+
+Non-goals:
+
+- We do not try to develop a localization system that
+  - does not fail in any environment
+  - works outside of the pre-defined ODD (Operational Design Domain)
+  - has unnecessarily better performance than required in autonomous driving
```

Comment:
```suggestion
- This design document does not aim to develop a localization system that
  - does not fail in any environment
  - works outside of the pre-defined ODD (Operational Design Domain)
  - has unnecessarily better performance than is required for autonomous driving
```

Reply:
It was more about trying to avoid a double negative ("...does not aim to.... does not fail") but if it changes the meaning then I can revert that first bullet point back to its original.

Reply:
OK. I think your suggestion `is infallible in all environments` is better than the original.